plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -42,16 +43,93 @@ global proc int mayaUsdTranslatorImport (string $parent,
 //
 {
     int $bResult = 1;
+    int $cw1 = 180;
+    int $cw2 = 300;
 
     if ($action == "post") {
         setParent $parent;
 
-		text -label "Maya USD import options to be determined.";
+        columnLayout -adj true -rs 5 mayaUsdTranslator_OptsCol;
+
+        rowLayout -numberOfColumns 3 -cw 1 $cw1 -cat 1 "right" 0;
+            text -label "Scope and Variants: " -al "right";
+            textField -ed false -nbg true -text "Please select a file to enable this option." -w $cw2 mayaUsdTranslator_SelectFileField;
+            button -label "View Hierarchy" -c "string $usdFile = currentFileDialogSelection(); usdImportDialog $usdFile;" mayaUsdTranslator_ViewHierBtn;
+
+            // Hide the button by default.
+            button -e -visible false mayaUsdTranslator_ViewHierBtn;
+        setParent ..;
+
+//        checkBoxGrp -label "Load Payloads: " -cw 1 $cw1 mayaUsdTranslator_LoadPayloadsCheckBox;
+//
+//        optionMenuGrp -label "Coordinate System: " -cw 1 $cw1 mayaUsdTranslator_CoordSystemOptionMenu;
+//            menuItem -label "Local space";
+//            menuItem -label "Object space";
+//            menuItem -label "World space";
+//        optionMenuGrp -e -sl 1 mayaUsdTranslator_CoordSystemOptionMenu;
+//
+//        optionMenuGrp -label "Materials: " -cw 1 $cw1 mayaUsdTranslator_MaterialsOptionMenu;
+//            menuItem -label "None";
+//            menuItem -label "Display Colors";
+//            menuItem -label "USD Shade";
+//        optionMenuGrp -e -sl 2 mayaUsdTranslator_MaterialsOptionMenu;
+//
+//        checkBoxGrp -label "InstancedPrims: " -label1 "Convert to Instances" -cw 1 $cw1 mayaUsdTranslator_InstancedPrimsCheckBox;
+//
+//        optionMenuGrp -label "Include Metadata: " -cw 1 $cw1 mayaUsdTranslator_IncludeMetadataOptionMenu;
+//            menuItem "All";
+//            menuItem "None";
+//            menuItem "Custom";
+//        optionMenuGrp -e -sl 3 mayaUsdTranslator_IncludeMetadataOptionMenu;
+//
+//        textFieldGrp -label "Specify Fields: " -cw 1 $cw2 -cw 2 150 mayaUsdTranslator_SpecifyFields;
+//
+//        textFieldGrp -label "Prefix Name: " -cw 1 $cw2 -cw 2 50 mayaUsdTranslator_PrefixName;
+//
+//        optionMenuGrp -label "Include Custom Attributes: " -cw 1 $cw1 mayaUsdTranslator_IncludeCustomAttribOptionMenu;
+//            menuItem "All";
+//            menuItem "None";
+//            menuItem "Custom";
+//        optionMenuGrp -e -sl 2 mayaUsdTranslator_IncludeCustomAttribOptionMenu;
+//
+//        checkBoxGrp -label "Animation Data: " -cw 1 $cw1 mayaUsdTranslator_AnimDataCheckBox;
+//
+//        intFieldGrp -label "Custom Frame Range Start/End: " -nf 2 -cw 1 $cw2 mayaUsdTranslator_CustomFrameRange;
```

Comment:
The rest of the options area here (which I showed at demo). But are all commented for now as they aren't implemented yet.